### PR TITLE
Migrates TranslateHelper service

### PR DESCRIPTION
**Pull request by dianabarsan**
# Description

Migrates the `Translate` service, now called `TranslateHelperService` for lack of a better name.
Also Remove comment from HeaderComponent. it's fully migrated.
Also removes to-do comment from format-data-records service (see https://github.com/medic/angular10-migration/issues/54). 
Also removes some not-needed or already migrated js files. 
Migrates SenderComponent unit tests. 

# Code review items

- Readable: Concise, well named, follows the [style guide](https://docs.communityhealthtoolkit.org/contribute/code/style-guide/), documented if necessary.
- Documented: Configuration and user documentation on [cht-docs](https://github.com/medic/cht-docs/)
- Tested: Unit and/or e2e where appropriate
- Internationalised: All user facing text
- Backwards compatible: Works with existing data and configuration or includes a migration. Any breaking changes documented in the release notes.

# License

The software is provided under AGPL-3.0. Contributions to this project are accepted under the same license.

### CODE REVIEW

**Comment by dianabarsan** [comment]
hey @latin-panda . 

I was planning this PR to be just removed code, but I somehow managed to squeeze something in. I decided to migrate the "Translate" service because bugs :D this way it's easier and tested. 

Could you, please, have a look? Thanks for your time!

File: webapp/src/ts/modals/edit-user/edit-user-settings.component.ts [hunk 1]
```diff
@@ -1,10 +1,11 @@
 import { Component, Input, OnInit } from '@angular/core';
-import { EditUserAbstract } from './edit-user.component';
-import {BsModalRef} from 'ngx-bootstrap/modal';
-import {UserSettingsService} from '@mm-services/user-settings.service';
-import {UpdateUserService} from '@mm-services/update-user.service';
-import {LanguagesService} from '@mm-services/languages.service';
-import {SetLanguageService} from '@mm-services/language.service';
+import { BsModalRef } from 'ngx-bootstrap/modal';
```
Comment (latin-panda) [inline webapp/src/ts/modals/edit-user/edit-user-settings.component.ts:2]: ✨

File: webapp/tests/karma/ts/components/sender.component.spec.ts [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+            }
+          }
+        }
+      }
+    };
+
+    fixture.detectChanges();
+    await fixture.whenStable();
+
+    expect(getElement('div .name').innerText).to.equal('Clark ');
```
Comment (latin-panda) [inline webapp/tests/karma/ts/components/sender.component.spec.ts:83]: In the future (after migration) I think we need to incorporate more tests like this one, where we check the template.

**Review by latin-panda — APPROVED** [review]
Nice and clean!! ☃️

File: webapp/tests/karma/ts/components/sender.component.spec.ts (hunk 2 above)
Comment (dianabarsan) [reply webapp/tests/karma/ts/components/sender.component.spec.ts:83]: yes <3<3<3 I actually quite enjoy writing these!

**Pull request merged**